cartography/intel/aws/ec2.py

```diff
@@ -496,6 +536,74 @@ def load_ec2_auto_scaling_groups(neo4j_session, data, region, current_aws_accoun
                 )
 
 
+def load_load_balancer_v2s(neo4j_session, data, region, current_aws_account_id, aws_update_tag):
+    ingest_load_balancer_v2 = """
+    MERGE (elbv2:LoadBalancerV2{id: {ID}})
+    ON CREATE SET elbv2.firstseen = timestamp(), elbv2.createdtime = {CREATED_TIME}
+    SET elbv2.lastupdated = {aws_update_tag}, elbv2.name = {NAME}, elbv2.dnsname = {DNS_NAME},
+    elbv2.canonicalhostedzonenameid = {HOSTED_ZONE_NAME_ID},
+    elbv2.type = {ELBv2_TYPE},
+    elbv2.scheme = {SCHEME}, elbv2.region = {Region}
+    WITH elbv2
+    MATCH (aa:AWSAccount{id: {AWS_ACCOUNT_ID}})
+    MERGE (aa)-[r:RESOURCE]->(elbv2)
+    ON CREATE SET r.firstseen = timestamp()
+    SET r.lastupdated = {aws_update_tag}
+    """
+    for lb in data['LoadBalancers']:
+        load_balancer_id = lb["DNSName"]
+
+        neo4j_session.run(
+            ingest_load_balancer_v2,
+            ID=load_balancer_id,
+            CREATED_TIME=str(lb["CreatedTime"]),
+            NAME=lb["LoadBalancerName"],
+            DNS_NAME=load_balancer_id,
+            HOSTED_ZONE_NAME_ID=lb.get("CanonicalHostedZoneNameID", ""),
```

Comment:
I'm in favor of changing this to 
`HOSTED_ZONE_NAME_ID=lb.get('CanonicalHostedZoneNameID')` so that it returns None instead of empty string if the field does not exist.  The neo4j python driver knows how to deal with None values: it simply does not add the field to the node.

Same comment applies to all the other empty string assignments below this one.

Reply:
Done :).